theme/layout/breakpoints.less

```diff
@@ -1,6 +1,10 @@
 // Breakpoint
 @responsive-breakpoint: 740px;
 
+@mobile: %(~"only screen and (max-width: %s)", (@responsive-breakpoint - 1));
+
+@desktop: %(~"only screen and (min-width: %s)", @responsive-breakpoint);
+
 .smallDeviceOnly(@rules) {
```

Comment:
Sorry to nitpick this, but could you add a comment that the mixins are for backwards compatibility?